src/page/shipping-step.page.ts

```diff
@@ -6,7 +6,7 @@ export class ShippingStepPage {
 
   constructor () {
     this.termsOfService = $('#cgv');
-    this.proceedToCheckout = $('#form > p > button > span');
+    this.proceedToCheckout = $('form .cart_navigation > button[type="submit"]');
```

Comment:
could you try to use a different attribute than type? for example, name attribute, will not be susceptible to changes and give us more context about what we are looking for

Reply:
Done